aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +38,46 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## January 2023
+
+- **Page Templates**
+  - Began identifying and exploring a variety of different types of common PageLayouts to create Page Templates. Coming soon!
+
+- **HPE Global Header**
+  - [Global Header](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?t=rcDz07wh34Xxtsft-0) available in Figma now.
+
+- **Feedback**
+  - [Guidance](https://design-system.hpe.design/templates/user-feedback-collection?q=feed) was added to the User Feedback Collection page.
+
+- **Figma Refactored Components:**
+  - [CheckBox](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=2940%3A62424&t=da2J4GDdKdAi7PIN-1)
+
+  - [CheckBoxGroup](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5732%3A78041&t=da2J4GDdKdAi7PIN-1)
+
+  - [RadioButton](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=3520%3A83485&t=da2J4GDdKdAi7PIN-1)
+
+  - [RadioButtonGroup](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5733%3A92878&t=da2J4GDdKdAi7PIN-1)
+
+  - [Tip](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5248%3A73430&t=da2J4GDdKdAi7PIN-1)
+
+  - [Feedback](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5554%3A77765&t=da2J4GDdKdAi7PIN-1)
+
+
+
+
```

Comment:
I think we can remove the extra spaces here

Reply:
resolved